kerastuner/engine/base_tuner.py

```diff
@@ -52,11 +55,18 @@ def __init__(self,
                  directory=None,
                  project_name=None,
                  logger=None,
-                 tuner_id=None):
+                 tuner_id=None,
+                 reload=True):
```

Comment:
`reload` is a reserved Python keyword. What are some other names we could use?

It isn't critical, we could still go with an argument named `reload`, but if we can avoid it we should.

Reply:
Hm, would `load_existing` be better?

Reply:
Alternatively: `overwrite=False`. What do you think?

Reply:
SGTM, done. That makes sense, and with that language we can simply delete the `project_dir` if `overwrite=True` is passed, which will avoid having users train two separate projects into the same project_name and then have it fail when they try to reload them